requirements/requirements-no-engine.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
   - sqlalchemy>=2.0.0
   - pandas-gbq>=0.19.0
   - pytables>=3.8.0
+  - c-blosc2<=2.14.1
```

Comment:
#7167 was merged so should we exclude these changes from this PR?

Reply:
yes, rebased